GE09-TEAM6/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Commit to using either double quotes (") or single quotes (') in all of your manifests.